README.md

```diff
@@ -1,56 +1,34 @@
-# Ktor CLI
-Ktor CLI is a command-line tool that brings the capability to create and run [Ktor](https://ktor.io/) applications from the command line.
-
-## Install CLI tools
-
-### macOS
-
-You can install Ktor CLI tools on macOS using [Homebrew](https://brew.sh/) as follows:
-1. Add a Ktor repository using the `brew tap` command:
-   ```
-   brew tap ktorio/ktor
-   ```
-2. Install Ktor CLI tools using `brew install`:
-   ```
-   brew install --build-from-source ktor\
-   ```
-
-### Linux
-
-On Linux, you can install Ktor CLI tools using [snap](https://snapcraft.io/):
+# Ktor-cli
+The tool allows generating [Ktor](https://ktor.io/) applications through the command line interface.
 
+## Building
+To build an executable issue the following command in the root directory of the repository:
+```shell
+go build github.com/ktorio/ktor-cli/cmd/ktor
 ```
-snap install --beta --classic ktor
-```
-
-### Available commands
-You can get a list of Ktor commands available to you by typing `ktor --help`.
-
-
-## Create a project
-
-To create a new Ktor project, pass a project name to the `ktor generate` command:
 
-```
-ktor generate ktor-sample
+If the build is successful the `ktor` executable should appear in the current directory.
+Also, the `go` command can be issued through Docker:
+```shell
+docker run --rm -v "$PWD":/usr/src/myapp -w /usr/src/myapp golang:1.21 go build -buildvcs=false -v github.com/ktorio/ktor-cli/cmd/ktor
 ```
 
-This command generates a simple Ktor project that uses the Gradle build system with Kotlin DSL.
+## Running
+To run the tool without an intermediate build, execute the following command:
+```shell
+go run github.com/ktorio/ktor-cli/cmd/ktor # followed by CLI args
+```
```

Comment:
It might be nice to clarify here that `go run github.com/ktorio/ktor-cli/cmd/ktor` replaces `ktor` as seen in the following chapter.